openssl/src/x509/mod.rs

```diff
@@ -78,6 +78,29 @@ impl X509StoreContextRef {
     pub fn error_depth(&self) -> u32 {
         unsafe { ffi::X509_STORE_CTX_get_error_depth(self.as_ptr()) as u32 }
     }
+
+    pub fn get_chain(&self) -> Option<&mut StackRef<X509Ref>> {
```

Comment:
We probably don't want a mutable reference. I believe the return type here will end up as `Option<&Ref<Stack<X509>>`.